.circleci/config.yml

```diff
@@ -1,13 +1,48 @@
-version: 2
+version: 2.1
+orbs:
+  go: circleci/go@1.7.1
+workflows:
+  main:
+    jobs:
+      - build
+
 jobs:
   build:
-    working_directory: /go/src/github.com/ahoy-cli/ahoy
-    docker:
-      - image: circleci/golang
+    executor:
+      name: go/default
+      tag: '1.18'
+    working_directory: /home/circleci/go/src/github.com/ahoy-cli/ahoy
     steps:
       - checkout
-      - run: git clone https://github.com/bats-core/bats-core.git && cd bats-core && sudo ./install.sh /usr/local && bats --version && cd -
-      - run: sudo apt install php
-      - run: go get -u golang.org/x/lint/golint
-      - run: bash build.sh -v
-      - run: ./ahoy test
+      - run:
+          name: Install BATS.
+          command: git clone https://github.com/bats-core/bats-core.git && cd bats-core && sudo ./install.sh /usr/local && bats --version && cd -
+      - run:
+          name: Install Go linter.
+          command: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.46.2
+      - run:
+          name: Install PHP for testing.
+          command: sudo apt update && sudo apt install php7.4
+      - go/load-cache
+      - go/mod-download
+      - run:
+          name: Get Go modules.
+          command: |
+            go mod tidy
+            go mod vendor
+            go mod verify
+      - go/save-cache
+      - run:
+          name: Build project.
+          command: |
+            make
+            make version
+      - run:
+          name: Lint code.
+          command: /home/circleci/go/bin/golangci-lint run || true
```

Comment:
@ocean I've re-introduced the linting with `||true` as it is failing - the code needs to be updated, but I do not want to introduce more changes in this PR. We can tackle this separately.

Reply:
Cool, good idea.

And nice neatening 👍